packages/devtools_app/lib/src/screens/memory/panes/diff/controller/diff_pane_controller.dart

```diff
@@ -61,6 +62,9 @@ class DiffPaneController {
   }
 
   Future<void> clearSnapshots() async {
+    for (var i = 0; i < snapshots.value.length; i++) {
```

Comment:
nit: use `for (final snapshot in snapshots)` to make this less verbose

Reply:
The index should start with 1 here. Fixed.